apps/src/utils.js

```diff
@@ -976,3 +976,22 @@ export function getAlphanumericId() {
   }
   return id.join('');
 }
+
+// These limits are based on the empirical observation that no data_json column
```

Comment:
[We know the datatypes of the Redshift table columns](https://github.com/code-dot-org/code-dot-org/pull/55828#issuecomment-1899387308) where these fields are ultimately saved, so we can be more explicit that we're choosing maximum sizes that are below the maximum lengths of the destination.

Reply:
🙌🏼 updated this! Thanks!